core/src/main/scala/org/bitcoins/core/wallet/utxo/AddressTagDb.scala

```diff
@@ -1,12 +1,6 @@
-package org.bitcoins.wallet.models
+package org.bitcoins.core.wallet.utxo
```

Comment:
Since these are part of the API shouldn't they be in 

`org.bitcoins.core.api.wallet` ?